### PR TITLE
Em 91

### PR DESCRIPTION
**Pull request by idan-matrix**
You can merge only this PR because I checkout from EM-92

### CODE REVIEW

File: src/Components/Forms/DiagnosisAndRecommendations/RecommendationsOnRelease /index.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+          inputRef={register}
+          label={t('Instructions for further treatment')}
+          width='45%'
+          multiline
+          InputLabelProps={{
+            shrink: instructionsForFurtherTreatment ? true : false,
+          }}
+          disabled={permission === 'view' ? true : false}
+        />
+        <StyledButton
```
Comment (matrix-amiel) [inline src/Components/Forms/DiagnosisAndRecommendations/RecommendationsOnRelease /index.js:64]: replace `StyledButton`  with new component in the asset for "select template" - `StyledSelectTempleteButton`
all the buttons of select template contain the same style and same title 
why we need to copy
```
  width='113px'
          height='32px'
          color='primary'
          margin='0 16px'
          variant='outlined'
          size='small'
```
In all the buttons

File: src/Components/Generic/PopupComponents/PopUpFormTemplates/index.js [hunk 2]
```diff
@@ -69,29 +70,63 @@ const PopUpFormTemplates = ({
...
 
-  const handleCreateData = async () => {
-    const templatesServerData = [];
-    let response = await getFormTemplates(
-      encounter.serviceTypeCode,
-      encounter.examinationCode.toString(),
-      formID,
-      formFields,
-    );
+  // const handleCreateData = async () => {
```
Comment (matrix-amiel) [inline src/Components/Generic/PopupComponents/PopUpFormTemplates/index.js:73]: Remove it if not needed

File: src/Components/Generic/PopupComponents/PopUpFormTemplates/index.js [hunk 3]
```diff
@@ -69,29 +70,63 @@ const PopUpFormTemplates = ({
...
+  //       formID,
+  //       formFields,
+  //     );
 
-    if (response.data && response.data.length > 0) {
-      for (let i = 0; i < response.data.length; i++) {
-        templatesServerData.push({ title: response.data[i] });
-      }
-    }
+  //     if (response.data && response.data.length > 0) {
```
Comment (matrix-amiel) [inline src/Components/Generic/PopupComponents/PopUpFormTemplates/index.js:83]: Remove it if not needed

File: src/Components/Generic/PopupComponents/PopUpFormTemplates/index.js [hunk 4]
```diff
@@ -69,29 +70,63 @@ const PopUpFormTemplates = ({
...
+  //       for (let i = 0; i < response.data.length; i++) {
+  //         templatesServerData.push({ title: response.data[i] });
+  //       }
+  //     }
 
-    if (templatesServerData && templatesServerData.length > 0) {
-      setTemplates(templatesServerData);
-    }
-  };
+  //     if (templatesServerData && templatesServerData.length > 0) {
```
Comment (matrix-amiel) [inline src/Components/Generic/PopupComponents/PopUpFormTemplates/index.js:89]: Remove it if not needed

File: src/Components/Forms/DiagnosisAndRecommendations/RecommendationsOnRelease /index.js [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+      <StyledDivider />
+      <Grid
+        container
+        direction='row'
+        justify='flex-start'
+        alignItems='baseline'>
+        <CustomizedTextField
+          name='instructionsForFurtherTreatment'
+          inputRef={register}
+          label={t('Instructions for further treatment')}
```
Comment (matrix-amiel) [inline src/Components/Forms/DiagnosisAndRecommendations/RecommendationsOnRelease /index.js:56]: for all the inputs with "select template" button, better to create const that contains the title of the input for example - t('Instructions for further treatment')
and use the variable for the label and for the first parameter in the handlePopUpProps.

now we have the same string in two places-
```
 label={t('Instructions for further treatment')}

 handlePopUpProps(
              t('Instructions for further treatment'),
```

It's low issue but more professional

File: src/Assets/Elements/StyledButton/index.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import { Button } from '@material-ui/core';
+import styled from 'styled-components';
+
+export const StyledButton = styled(Button)`
```
Comment (matrix-amiel) [inline src/Assets/Elements/StyledButton/index.js:4]: is this the default style for a button in all the screens?
If yes let's notify all the developers about this component

File: src/Components/Generic/PopupComponents/PopUpFormTemplates/index.js (hunk 2 above)
Comment (idan-matrix) [reply src/Components/Generic/PopupComponents/PopUpFormTemplates/index.js:73]: True will delete that

File: src/Components/Forms/DiagnosisAndRecommendations/RecommendationsOnRelease /index.js (hunk 1 above)
Comment (idan-matrix) [reply src/Components/Forms/DiagnosisAndRecommendations/RecommendationsOnRelease /index.js:64]: Ok

File: src/Assets/Elements/StyledButton/index.js (hunk 6 above)
Comment (idan-matrix) [reply src/Assets/Elements/StyledButton/index.js:4]: Idk if this is the default button for everyone it is for me. we need to discuss about it if they are using something else and if we can combine it to one component

File: src/Components/Forms/DiagnosisAndRecommendations/DiagnosisAndTreatment/index.js [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+      <StyledDivider />
+      <Grid
+        container
+        direction='row'
+        justify='flex-start'
+        alignItems='baseline'>
+        <CustomizedTextField
+          name='findingsDetails'
+          inputRef={register}
+          label={t('Findings details')}
```
Comment (matrix-amiel) [inline src/Components/Forms/DiagnosisAndRecommendations/DiagnosisAndTreatment/index.js:59]: Sorry, it was not clear, I want we move the field name to constant for all the inputs with StyledSelectTemplateButton.

Here also

File: src/Components/Forms/DiagnosisAndRecommendations/DiagnosisAndTreatment/index.js [hunk 8]
```diff
@@ -0,0 +1,151 @@
...
+      </Grid>
+      <Grid
+        container
+        direction='row'
+        justify='flex-start'
+        alignItems='baseline'>
+        <CustomizedTextField
+          inputRef={register}
+          name='diagnosisDetails'
+          label={t('Diagnosis details')}
```
Comment (matrix-amiel) [inline src/Components/Forms/DiagnosisAndRecommendations/DiagnosisAndTreatment/index.js:91]: move filed name to constant

File: src/Components/Forms/DiagnosisAndRecommendations/DiagnosisAndTreatment/index.js [hunk 9]
```diff
@@ -0,0 +1,151 @@
...
+      </Grid>
+      <Grid
+        container
+        direction='row'
+        justify='flex-start'
+        alignItems='baseline'>
+        <CustomizedTextField
+          inputRef={register}
+          name='treatmentDetails'
+          label={t('Treatment details')}
```
Comment (matrix-amiel) [inline src/Components/Forms/DiagnosisAndRecommendations/DiagnosisAndTreatment/index.js:123]: move filed name to constant

File: src/Components/Forms/DiagnosisAndRecommendations/DrugRecommendation/index.js [hunk 10]
```diff
@@ -0,0 +1,297 @@
...
+                  }}
+                  label={t('To date')}
+                  width='30%'
+                />
+              </Grid>
+              <Grid container direction='row' alignItems='baseline'>
+                <CustomizedTextField
+                  disabled={checkIsDisabled('drugName', index)}
+                  name={`drugRecommendation[${index}].instructionsForTheDrug`}
+                  label={t('Instructions for the drug')}
```
Comment (matrix-amiel) [inline src/Components/Forms/DiagnosisAndRecommendations/DrugRecommendation/index.js:249]: move filed name to constant

**Pull request merged**